lib/rouge/lexers/rml.rb

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+    module Lexers
```

Comment:
**suggestion:** We use 2-space indentation in our code base. Would it be possible to fix all the indentations on this file? 🙏🏼